IMuaythai.Api/Validators/Contest/ContestRequestValidator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using FluentValidation;
+using IMuaythai.Models.Contests;
+
+namespace IMuaythai.Api.Validators.Contest
+{
+    public class ContestRequestValidator : AbstractValidator<ContestRequestModel>
+    {
+        public ContestRequestValidator()
+        {
+            RuleFor(x => x.Id).NotNull().WithMessage("Id can not be null");
+            RuleFor(x => x.AcceptanceDate).NotNull().WithMessage("AcceptanceDate can not be null");
+            RuleFor(x => x.ContestId).NotNull().WithMessage("ContestId can not be null");
+            RuleFor(x => x.IssueDate).NotNull().WithMessage("IssueDate can not be null");
+            RuleFor(x => x.Status).NotNull().WithMessage("Status can not be null");
+            RuleFor(x => x.Type).NotNull().WithMessage("Type can not be null");
+            RuleFor(x => x.AcceptedByUserId).Length(0, 450).WithMessage("AcceptedByUserId is too long");
```

Comment:
it can be null, can you check what happened when AcceptedByUserId will be null. Will it work?

Reply:
Yep it may cause problem if null, added "Length(0, 450).When(x => x.AcceptedByUserId != null)" so if AcceptedByUserId is null this rule will be ignored

Reply:
ok, so let's add this Where